CFC_DataCollector/recommender/Pipeline.py

```diff
@@ -0,0 +1,41 @@
+""" High level pipeline
+
+3 Pipelines
```

Comment:
I think we can probably delete this, since the individual pipelines have now been created and will probably be invoked at different cadences.